### PR TITLE
Enable stable and development versions

### PR DESCRIPTION
**Pull request by walkowif**
https://github.com/insightsengineering/idr-tasks/issues/717

### CODE REVIEW

**Review by danielinteractive — APPROVED** [review]
thanks a lot @walkowif !

File: _quarto-development.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+website:
+  title: Biomarker Analysis Catalog - Dev
```
Comment (walkowif) [inline _quarto-development.yml:2]: @danielinteractive Are these titles okay? Or should `- Dev` and `- Stable` be changed to something else?

File: _quarto-development.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  navbar:
+    right:
+      - text: "Dev"
```
Comment (walkowif) [inline _quarto-development.yml:5]: @danielinteractive Should the drop-down in the navbar (for switching versions) use `Dev` and `Stable` names?

File: _quarto-development.yml (hunk 1 above)
Comment (danielinteractive) [reply _quarto-development.yml:2]: yeah those are fine

File: _quarto-development.yml (hunk 2 above)
Comment (danielinteractive) [reply _quarto-development.yml:5]: is ok!

**Pull request merged**